pints/tests/test_toy_goodwin_oscillator_model.py

```diff
@@ -40,6 +41,48 @@ def test_values(self):
         self.assertAlmostEqual(values[100, 1], 0.1779547, places=6)
         self.assertAlmostEqual(values[100, 2], 2.6074527, places=6)
 
+    def test_sensitivity(self):
```

Comment:
Could you add a comment in each of these? I don't get what the difference between test_sensitivity and test_sensitivities is

Reply:
Good point -- I've combined these into a single test method as it didn't make sense to keep these separate.